packages/rich-text/src/component/use-anchor.js

```diff
@@ -45,13 +45,18 @@ export function useAnchor( { editableContentElement, value, settings = {} } ) {
 			return;
 		}
 
+		const selectionWithinEditableContentElement =
+			editableContentElement?.contains( selection?.anchorNode );
+
 		const range = selection.getRangeAt( 0 );
 
 		if ( ! activeFormat ) {
 			return {
 				ownerDocument: range.startContainer.ownerDocument,
 				getBoundingClientRect() {
-					return range.getBoundingClientRect();
+					return ! selectionWithinEditableContentElement
```

Comment:
nit: can we start with 'yes'? I mean remove the negation and reverse the ternary?

Reply:
✅